include/cpu.h

```diff
@@ -39,13 +39,15 @@
 
 #define CPU_CYCLES_LOWER_LIMIT		200
 
-
-#define CPU_ARCHTYPE_MIXED			0xff
-#define CPU_ARCHTYPE_386SLOW		0x30
-#define CPU_ARCHTYPE_386FAST		0x35
-#define CPU_ARCHTYPE_486OLDSLOW		0x40
-#define CPU_ARCHTYPE_486NEWSLOW		0x45
-#define CPU_ARCHTYPE_PENTIUMSLOW	0x50
+#define CPU_ARCHTYPE_MIXED       0xff
+#define CPU_ARCHTYPE_086         0x05
+#define CPU_ARCHTYPE_186         0x15
+#define CPU_ARCHTYPE_286         0x25
+#define CPU_ARCHTYPE_386SLOW     0x30
+#define CPU_ARCHTYPE_386FAST     0x35
+#define CPU_ARCHTYPE_486OLDSLOW  0x40
+#define CPU_ARCHTYPE_486NEWSLOW  0x45
+#define CPU_ARCHTYPE_PENTIUMSLOW 0x50
 
```

Comment:
Maybe a nice application of enum? This jumped out at me visually, with the longer list:

![2023-04-07_14-48](https://user-images.githubusercontent.com/1557255/230683320-7e7fb764-f6f3-4eaa-ab0e-9c0abfe38f5e.png)

And can even keep the exact hex values (if they're meaningful)

Reply:
Changed, in a separate commit. Note: this was in 9% a search&replace adaptation; there was no more cleanup in the CPU code. This is an XMS improvement PR after all :)